src/main/java/task/Event.java

```diff
@@ -0,0 +1,25 @@
+package task;
+
+public class Event extends Task {
+    protected String from;
+    protected String to;
```

Comment:
Perhaps the variable names could be improved since they are referring to a specific time. You could consider using startTime and endTime.